azure-toolkit-libs/azure-toolkit-springcloud-lib/src/main/java/com/microsoft/azure/toolkit/lib/springcloud/SpringCloudApp.java

```diff
@@ -163,4 +163,9 @@ public SpringCloudPersistentDisk getPersistentDisk() {
     public boolean isPersistentDiskEnabled() {
         return Objects.nonNull(this.getPersistentDisk());
     }
+
+    @Override
+    public ServiceLinkerModule getServiceLinkerModule() {
+        return linkerModule;
```

Comment:
this method is not needed if this field is renamed as `linkerModule`, because this class has `@Getter`.